getting-started/first-steps-sl.md

```diff
@@ -0,0 +1,93 @@
+---
+title: First Steps
+page_title: First Steps
+description: First Steps
+slug: getting-started-first-steps-sl
+tags: get,started,first,steps
+published: True
+position: 0
+---
+
+# First Steps
+
+This article explains how to get the Telerik UI for {{ site.framework_name }} controls in your project and start using them quickly.
```

Comment:
This article seems like it was created only for Silverlight. In this case you don't need the 'framewor_kname' tag here.